include/revng/Pipeline/Target.h

```diff
@@ -191,7 +151,24 @@ class TargetsList {
 
 public:
   TargetsList() = default;
-  TargetsList(List C) : Contained(std::move(C)) { removeDuplicates(); }
+  TargetsList(List C) : Contained(std::move(C)) {}
+  static TargetsList allTargets(const Context &Ctx, const Kind &K) {
+    TargetsList ToReturn;
+    K.allTargets(Ctx, ToReturn);
+    return ToReturn;
+  }
+
+  bool operator==(const TargetsList &Other) const {
```

Comment:
Why do you need to implement this?

Reply:
contracts sometimes needs to check if a target list is exactly the one they expected. These lists are sorted, so it is like set equivalence.

Reply:
How is this different from the `= default`?

Reply:
forgot it was possible to do in cpp20